content/posts/redis-modules-into-arm-land-part1.md

```diff
@@ -0,0 +1,275 @@
+
+- Get a microSD Card: 64GB, Class 10, anything comparable (that is, at the same price level) to Samsung EVO or SanDisk Ultra will do.
+- Download the OS [image](http://cdimage.ubuntu.com/releases/19.04/release/ubuntu-19.04-preinstalled-server-arm64+raspi3.img.xz) (typically .img or .raw format, archived) and extract it. While it is possible to install from an .iso file, using .img images is much more straightforward and therefore recommended.
+- Write it to a microSD card with a tool like [Balena Etcher](https://www.balena.io/etcher/) (it's available for all platforms, I personally use [Win32 Disk Imager](https://sourceforge.net/projects/win32diskimager/)). In order to write, you should use the build-in microSD reader in your laptop or get an external one with a USB interface.
+- Insert the microSD into the RPi and power the device on.
+- Username/password: ubuntu/ubuntu
+
+#### Installing Raspbian Buster on RPi 3 & 4
+
+* Repeat the above steps, with Raspbian Lite [image](https://downloads.raspberrypi.org/raspbian_lite_latest).
+* Username/password: pi/raspberry
+
+#### Connecting to Workstation
+
+By "workstation" I refer to a Linux or macOS host, that holds one's development environment and git repositories. As a side note,  I warmly recommend using a desktop PC (that's another blog post), though most people use laptops. In either case, I also recommend having some virtualization infrastructure on your workstation, VMware Workstation/Fusion or VirtualBox being good solutions.
+
+So, we need to establish a network connection between the RPi and the workstation. As mentioned before, RPi 4 has a 1Gbps NIC, which is a great improvement over the RPi 3 with its 100Mbps NIC. Connecting the RPi to a network is simple: get any gigabit Ethernet unmanaged switch and two Ethernet cables, then hook the RPi to the switch and connect the switch to your gateway Ethernet port. You can hook your workstation to the switch as well.
+
+At this stage, we need to gather some information from the workstation. 
+
+First, we need to determine UID & GID of the workstation user that owns the views:
+
+```
+id
+```
+
+We'll call them MY-UID & MY-GID.
+
+Next, we need to find out what's our time zone
+
+```
+timedatectl
+```
+
+We'll call it MY-TIMEZONE.
+
+Finally, we'll need your workstation's IP:
+
+```
+ip a
+```
+
+We'll call if MY-WORKSTATION-IP.
+
+#### RPi Configuration
+
+During the initial setup, you'll need to have the RPi connected to a monitor and a keyboard. Once setup is done, it can be controlled from your workstation via SSH.
+
+Another good practice is to avoid cloning source code into the RPi, but rather share it between your workstation and RPi via NFS.

```

Comment:
Change to: "Another good practice is to avoid cloning source code into the RPi, instead sharing it between your workstation and RPi via NFS."